lib/twitter_cldr/js/renderers/numbers/rbnf.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module Numbers
+        module RBNF
+
+        	class RBNFRenderer < TwitterCldr::Js::Renderers::Base
```

Comment:
Besides, the class name doesn't match the file name. And do we need `RBNF` in the class name with it being inside of `RBNF` module?